File: 백준/Bronze/2562. 최댓값/최댓값.cc
#include <iostream>

using namespace std;

int main(void) {
    int num[10], index = 0, max = 0;
    
    for (int i = 0; i < 9; i++) {
        cin >> num[i];
        if (max < num[i]) {
            max = num[i];
            index = i;
        }
    }
    cout << num[index] << "\n" << index + 1 << endl;
    return 0;
}

File: Java/백준/Bronze/2562. 최댓값/최댓값.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        final int len = 9;
        int max = 0, num, maxIndex = 0;
        
        for (int i = 0; i < len; i++) {
            num = s.nextInt();
            if (num > max) {
                max = num;
                maxIndex = i + 1;
            }
        }
        System.out.printf("%d\n%d", max, maxIndex);
        s.close();
    }
}